spec/acceptance/suites/default/redis_multi_instances_one_host_spec.rb

```diff
@@ -34,29 +46,31 @@ class { 'redis':
     it { is_expected.to be_installed }
   end
 
-  describe service('redis-server-redis1') do
-    it { is_expected.to be_running }
+  describe service(redis_name) do
```

Comment:
Could you add `not_to be_running` as well?

Reply:
Sure, will do.